### PR TITLE
Don't run update on dead keys

### PR DESCRIPTION
**Pull request by kesla**
@ellell Can you try and help out w a test for this? I can't figure it out so I'm pretty stuck.

My thesis righ now is that since this is a pretty new feature to Chrome (Chrome 51) maybe electron is behind?

### CODE REVIEW

**Comment by kesla** [comment]
The functionality works really well fwiw

**Comment by ellell** [comment]
@kesla yup, taking a look now.

File: test/embeds-test.js [hunk 1]
```diff
@@ -116,8 +118,10 @@ test('InstagramEmbed - onResize', t => {
   const el = document.body.appendChild(document.createElement('div'));
   const expectedHeight = 754;
   const onResize = ({height}) => {
-    t.equal(height, expectedHeight);
-    t.end();
+    setTimeout(() => {
+      t.equal(height, expectedHeight);
+      t.end();
+    }, 0);
```
Comment (kesla) [inline test/embeds-test.js]: perhaps comment or so about why we need this? Can easily be removed since it looks unnecessary but is needed...

Comment (ellell) [inline test/embeds-test.js]: yup

**Pull request merged**